packages/utils/src/options.ts

```diff
@@ -1,64 +1,77 @@
+import { Provider } from "@tko/provider";
+
+interface CustomBindingGlobalProperties {
+  String;
```

Comment:
unclear what `String` is

Reply:
It was wrong, fixed it https://github.com/Auge19/tko/pull/4

```typescript
interface CustomBindingGlobalProperties {
  [customBindingName: string]: any;
}
``` 

`String` and `isObservable` are only used as test-case-props in `templatingBehaviors.ts`

Reply:
Fixed in https://github.com/Auge19/tko/pull/4